graylog2-server/src/main/java/org/graylog2/database/DbEntity.java

```diff
@@ -34,5 +34,5 @@
     String NO_TITLE = "";
 
     String ALL_ALOWED = "";
-    String NOBODY_ALOWED = "youshall:notpass";
+    String NOBODY_ALOWED = "__no_access__";
```

Comment:
Sorry for being the party :hankey:. :smile:

Reply:
I will come back as Gandalf the White! ;)